app/mailers/rock_pebble_mailer.rb

```diff
@@ -0,0 +1,56 @@
+class RockPebbleMailer < ApplicationMailer
+  def confirmation_message(info)
+    address = info[:rock_email]
+    name = info[:pebble_name]
+    @message = "#{name} has requested you as a Rock through the Rock and Pebble
+                Program on Paired.tech. Please visit #{paired_link} to accept or deny
+                this request."
+    mail(to: address, subject: "Someone has requested you as a Rock!")
+  end
+
+  def rock_accepted_message(info)
+    address = info[:pebble_email]
+    name = info[:rock_name]
+    @message = "#{name} has accepted your Rock request. Please use their contact
+                information provided in your Paired Rock and Pebble dashboard
```

Comment:
On this line, I'd suggest changing "your Paired Rock and Pebble dashboard" to "your Rock & Pebble dashboard at https://www.paired.tech" so they have an immediate link to follow. (Maybe we should add a link to Paired to the overall email template as well so it's always there?)